src/api/controllers/articlesController.js

```diff
@@ -108,10 +108,58 @@ class articlesController {
     const { slug } = req.params;
 
     // @delete an article in Database
-    await article.destroy({ where: { slug } });
+    await Article.destroy({ where: { slug } });
 
     // @returning the response
     res.status(200).send({ message: 'Article deleted successfully!' });
   }
+
+  static async likeArticle(req, res) {
+    const { slug } = req.params;
+
+    // Find the article
+    const query = await Article.findAll({ where: { slug } });
+
+    if (!query[0]) {
+      return res.status(404).json({ message: `Article with slug: ${slug} not found` });
+    }
+
+    const { dataValues: foundArticle } = query[0];
+
+    // Increment like field by one
+    foundArticle.likes += 1;
+
+    // Update table
+    await Article.update(
+      foundArticle,
+      { where: { slug } }
+    );
+
+    return res.status(200).json({ message: 'OK' });
```

Comment:
Update the response message to something like, `Article Liked.`

Reply:
Updated.